euth/documents/api.py

```diff
@@ -10,5 +10,5 @@ class DocumentViewSet(viewsets.ModelViewSet):
     queryset = Document.objects.all()
     serializer_class = DocumentSerializer
     permission_classes = (
-        IsModerator,
+        permissions.IsAuthenticatedOrReadOnly, IsModerator,
```

Comment:
This is fixing an exception in the moderator exception if the user isn't logged in? Can this be fixed in the `isModerator` permission?